addons/mail/models/mail_render_mixin.py

```diff
@@ -271,6 +271,7 @@ def _render_eval_context(self):
             'is_html_empty': is_html_empty,
             'slug': self.env['ir.http']._slug,
             'user': self.env.user,
+            'env': self.env,
```

Comment:
Help @tde-banana-odoo :pray: 

Need your opinion here so we can merge this one please.

Reply:
ping @tde-banana-odoo